runtime/docker/docker.go

```diff
@@ -183,28 +189,29 @@ func (c *DockerRuntime) CreateNet(ctx context.Context) (err error) {
 
 // DeleteNet deletes a docker bridge
 func (c *DockerRuntime) DeleteNet(ctx context.Context) (err error) {
-	if c.Mgmt.Network == "bridge" {
-		log.Debug("Skipping potential deletion of docker default bridge 'bridge'.")
+	network := c.Mgmt.Network
+	if network == "bridge" || c.keepMgmtNet {
+		log.Infof("Skipping potential deletion of docker default bridge '%s'", network)
```

Comment:
lets make it like that maybe:

```go
network := c.Mgmt.Network
	if network == "bridge" || c.keepMgmtNet {
		log.Debugf("Skipping deletion of '%s' network", network)
```

Reply:
Changing the message makes absolute sense, but this debug level should be the same as the above.
Otherwiese that would be cleaning up networking, but that would be strange if it would then still exist ... 
so lets either do debug or info for both...